App.js

```diff
@@ -7,8 +7,11 @@ import LogoTitle from "./components/LogoTitle";
 import Welcome from "./screens/Welcome";
 import CurrentListings from "./screens/CurrentListings";
 import ListingDetails from "./screens/ListingDetails";
-import PostItem from "./screens/PostItem"
-import PaymentInstructions from "./screens/PaymentInstructions"
+import PostItem from "./screens/PostItem";
+import PaymentInstructions from "./screens/PaymentInstructions";
+import ChooseCharity from "./screens/ChooseCharity";
+import PostConfirmation from "./screens/PostConfirmation";
+
 
```

Comment:
The only imports I don't see, which I have locally, are 
import LogoTitle from "./components/LogoTitle"
import { StoreProvider } from "./store"

is the navigation container wrapped in a store provider on yours?

Reply:
LogoTitle is there if you scroll up and the nav container is wrapped in a store provider

Reply:
okay, perfect!